langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -333,6 +470,13 @@ public isolated function elementChildren(xml x, string? nm = ()) returns xml<Ele
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml books = xml `<book>Java</book><book>Ballerina</book>`;
+# books.map(function (xml xmlContent) returns xml {
```

Comment:
```suggestion
#  books.map(function (xml xmlContent) returns xml => 
     xml `<book kind="GuideBook">${xmlContent.children()}</book>`);
```
Can be applied to wherever the function body is just a return statement.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39130/commits/3a7088c41159f3c56a37443fa2036905f399eaa5